### PR TITLE
Fix highlighting problem in the console.

### PR DESCRIPTION
**Pull request by jasoryeh**
I kept the on click in case someone needed it. 
Problem 2 on #706

### CODE REVIEW

**Comment by DaneEveritt** [comment]
This should just remove `#terminal` from the list of elements to focus on, rather than disabling all of them.

File: public/themes/pterodactyl/js/frontend/console.js [hunk 1]
```diff
@@ -47,7 +47,7 @@ $(document).ready(function () {
 
     $terminalInput.focus();
     $('.terminal_input--prompt, #terminal_input, #terminal, #terminalNotify').on('click', function () {
```
Comment (lancepioch) [inline public/themes/pterodactyl/js/frontend/console.js]: Remove the whole statement instead of commenting it out.

Comment (DaneEveritt) [reply public/themes/pterodactyl/js/frontend/console.js]: @lancepioch see my comment in the actual PR about just removing `#terminal` from the list.

Comment (jasoryeh) [reply public/themes/pterodactyl/js/frontend/console.js]: Removing #terminal should work.

**Pull request merged**